tests/unittest/_torch/sampler/test_torch_sampler.py

```
@@ -635,231 +635,259 @@ def _uut(res=res):
+            self.finish_reasons = finish_reasons
+
+        def __repr__(self):
+            return f"RequestCase({self.prompt=}, {self.new_tokens=}, {self.finish_reasons=}, \
+            {self.request.max_new_tokens=}, {self.request.end_id=}, {self.request.stop_words_list=})"
+
+        @classmethod
+        def build(
+            cls,
+            requests: list["TestFinishReasons.RequestCase"],
+            *,
+            check_no_cuda_sync: bool = True,
```

Ruff (0.15.0)

[warning] Unused class method argument: `check_no_cuda_sync`

(ARG003)